tests/testthat/test-fims-estimation.R

```diff
@@ -167,7 +167,7 @@ test_that("deterministic test of fims", {
   sdr <- TMB::sdreport(obj)
   sdr_fixed <- summary(sdr, "fixed")
 
-  report <- obj$report()
+  report <- obj$report(obj$par)
```

Comment:
Perhaps include documentation here as a reminder for developers to provide the parameter list to obj$report.